main.go

```diff
@@ -326,7 +337,7 @@ func getBuilds(conf *config, pr *github.PullRequestEvent) []buildOptions {
 
 	makeQEMU := false
 
-	for _, watchRepo := range conf.watchRepositories {
+	for _, watchRepo := range conf.watchRepositoriesPipeline {
```

Comment:
Stretch, but maybe we should start thinking about splitting this up soon. In which case `config` would be a pretty self-contained package.

Reply:
Maybe next time ;)